src/hooks/useTranslate/useTranslate.ts

```diff
@@ -6,7 +6,7 @@ import i18n from "i18n-js";
 import { CampaignConfigsStoreContext } from "../../context/campaignConfigsStore";
 
 export type TranslationHook = {
-  c13nt: (key: string, campaignKey?: string, defaultString?: string) => string;
+  c13nt: (key?: string, campaignKey?: string, defaultString?: string) => string;
```

Comment:
Why is this optional?

Reply:
i was thinking that since we already have a fallback in case `key` is missing, `key` should be optional. 

also, because `selectedChoice` is optional thus would make  `src/components/CustomerQuota/ItemsSelection/IdentifierLayout/IdentifierSelectionInput.tsx line26` more straight forward